src/components/v2/devtronStackManager/DevtronStackManager.component.tsx

```diff
@@ -429,91 +431,214 @@ export const InstallationWrapper = ({
     upgradeVersion,
     isUpgradeView,
     isActionTriggered,
+    releaseNotes,
     updateActionTrigger,
+    showPreRequisiteConfirmationModal,
+    setShowPreRequisiteConfirmationModal,
+    preRequisiteChecked,
+    setPreRequisiteChecked,
 }: InstallationWrapperType): JSX.Element => {
     const history: RouteComponentProps['history'] = useHistory()
     const location: RouteComponentProps['location'] = useLocation()
     const latestVersionAvailable = isLatestVersionAvailable(serverInfo?.currentVersion, upgradeVersion)
+    const [preRequisiteList, setPreRequisiteList] = useState<
+        { version: string; prerequisiteMessage: string; tagLink: string }[]
+    >([])
+    useEffect(() => {
+        if (releaseNotes) {
+            const _preRequisiteList = []
```

Comment:
Can we make a separate function for this and then call it in useEffect?

Reply:
done